package.json

```diff
@@ -23,9 +23,11 @@
     "url": "https://github.com/cerner/terra-toolkit/issues"
   },
   "bin": {
-    "tt:aggregate-translations": "bin/aggregate-translations.js",
-    "tt:serve": "bin/serve.js",
-    "tt:serve-static": "bin/serve-static.js"
+    "tt-aggregate-translations": "bin/aggregate-translations.js",
+    "tt-clean-screenshots": "bin/clean-screenshots.js",
+    "tt-serve": "bin/serve.js",
+    "tt-serve-static": "bin/serve-static.js",
+    "tt-wdio": "bin/tt-wdio.js"
```

Comment:
I can confirm that these changes fix the issue windows was having with the bin about "tt:serve" not existing.